india_compliance/gst_india/client_scripts/party.js

```diff
@@ -70,6 +70,9 @@ function validate_gstin(doctype) {
             if (PAN_REGEX.test(pan)) {
                 frm.doc.pan = pan;
                 frm.refresh_field("pan");
+                if(["Customer", "Supplier"].includes(doctype)){
```

Comment:
Can be common condition for `set_party_type`. You can access doctype as `frm.doc.doctype`